examples/src/main/javascript/tor.js

```diff
@@ -6,19 +6,15 @@ var params = bcj.params.MainNetParams.get();
 var context = new bcj.core.Context(params);
 bcj.utils.BriefLogFormatter.init();
 
-var InetAddress = Java.type("java.net.InetAddress");
+var PeerAddress = Java.extend(Java.type("org.bitcoinj.core.PeerAddress")
```

Comment:
Java.extend is for when you want to subclass it. In this case it's no longer needed, so it can be just a plain call to Java.type